homeassistant/components/rainmachine/__init__.py

```diff
@@ -386,14 +389,16 @@ def device_class(self):
     def device_info(self):
         """Return device registry information for this entity."""
         return {
+            'config_entry_id': self.rainmachine.config_entry_id,
```

Comment:
This key is not pulled in through `device_info` and will be ignored, so remove it.  It's set automatically based on the config entry that added the device.  [See here for the code that processes `device_info`.](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity_platform.py#L289-L302)